### DISCUSSION TITLE
Question: Is it it supported on windows 10?

### DISCUSSION THREAD

**Discussion by Folaht** [vagrant-libvirt/vagrant-libvirt#1379] [General]
Is it possible to use vagrant-libvirt on a windows 10 OS?

![image](https://user-images.githubusercontent.com/9342090/135496092-0335c17a-8f08-4797-87f0-e672d20aec21.png)

**Comment by electrofelix** [comment]
I've not worked with libvirt on windows, but I understand that it's limited to only working as a client. Unfortunately I've also not done anything around installing the client and ruby and seeing if it's possible to have vagrant be able to correctly link against them. That said, I do see there are issues using various connection types.

I presume you want to use this to connect to a remote libvirt instance? Given that libvirt doesn't appear to support running on windows https://libvirt.org/windows.html.

So might have better success using the docker image with vagrant + vagrant-libvirt bundled together https://github.com/vagrant-libvirt/vagrant-libvirt/#using-the-container-image. Should be able to skip mounting in the libvirt folders and set the env var `IGNORE_MISSING_LIBVIRT_SOCK=1` to ignore the missing socket file.

I do have a new work laptop where it's likely that I'll get to try out docker on windows and WSL2 which will probably help work out whether there are special steps for getting the docker image, but might be a few weeks before I get to try taking vagrant-libvirt for a spin on it.

**Comment by Folaht** [comment]
> I presume you want to use this to connect to a remote libvirt instance? Given that libvirt doesn't appear to support running on windows https://libvirt.org/windows.html.

Yes. 

> So might have better success using the docker image with vagrant + vagrant-libvirt bundled together https://github.com/vagrant-libvirt/vagrant-libvirt/#using-the-container-image. Should be able to skip mounting in the libvirt folders and set the env var IGNORE_MISSING_LIBVIRT_SOCK=1 to ignore the missing socket file.

That sounds too complicated. I will offer windows users virtualbox then.
